docs/docs/zh/guide/installation/kubernetes.md

```diff
@@ -522,226 +522,227 @@ common:
 
 ## 附录-配置
 
-| Parameter                                                                         | Description                                                                                                                    | Default                                               |
```

Comment:
That's wired. Our Chinese docs for K8S seemed written in English?

Reply:
> That's wired. Our Chinese docs for K8S seemed written in English?

Yes, but only for the `Appendix-Configuration`.